api/pkg/apis/v1alpha1/utils/utils.go

```diff
@@ -323,6 +324,51 @@ func jsonPathQuery(obj interface{}, jsonPath string) (interface{}, error) {
 	}
 }
 
+func isAlphanum(query string) bool {
+	return regexp.MustCompile(`^[a-zA-Z0-9]+$`).MatchString(query)
```

Comment:
Can we do something like ^(?:\.|\[|\(|length|keys|flatten|map|select|unique|sort|join|add|reduce)

Reply:
As discussed offline, update to check if string is quoted by `` to determine if it is a jq expression.